src/test/resources/ci/scripts/setup.sh

```diff
@@ -17,17 +17,17 @@ OS=$(uname -s)
 
 # Semeru JDK version control constants.
 # Example:
-# Version (SEMERU_OPEN_JDK_BUILD): 17.0.4.1
-# OpenJDK (SEMERU_OPEN_JDK_BUILD + SEMERU_OPEN_JDK_BUILD): 17.0.4.1+1
-# OpenJ9  (SEMERU_OPENJ9_VERSION): 0.33.1
+# Version (SEMERU_OPEN_JDK_BUILD): 17.0.6
+# OpenJDK (SEMERU_OPEN_JDK_BUILD + SEMERU_OPEN_JDK_BUILD): 17.0.6_10
```

Comment:
Is it supposed to be `(SEMERU_OPEN_JDK_VERSION + SEMERU_OPEN_JDK_BUILD)`?

Reply:
Yep. will change it on the next drop.